Main.cs

```diff
@@ -139,32 +120,50 @@ private void GenerateThumb()
             }
 
             // 1) some setup
-            var tmpImage = (Image)cachedImage.Clone();
+            var tmpImage = new Bitmap(cachedImage.Width, cachedImage.Height);
+            var copyImage = (Image)cachedImage.Clone();
+
+            var backgroundColor = new SolidBrush(Color.FromArgb(42, 42, 42));
+            var backgroundEdgeColor = new Pen(Color.FromArgb(42, 42, 42));
+
+            int thumbWidth = tmpImage.Width;
+            int thumbHeight = tmpImage.Height;
+
+            const int margin = 75;

```

Comment:
margin should be % of either width or height?

Reply:
It's no problem. because, always get every thumbnails height 1,080 size.